containers/Dockerfile.buildenv

```diff
@@ -287,3 +280,103 @@ RUN wget https://downloadmirror.intel.com/684899/sde-external-9.0.0-2021-11-07-l
 
 # Remove the apt-get cache in order to reduce image size
 RUN apt-get -y clean
+
+FROM dev as ci
+
+# Install all the compilers we are going to test on CI
+RUN apt-get update -y \
+    && apt-get install -y gcc-10 g++-10 gfortran-10 \
+                          gcc-11 g++-11 gfortran-11
+
+# Install clang-11 and clang-13
+RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - \
+    && add-apt-repository 'deb http://apt.llvm.org/focal/ llvm-toolchain-focal-11 main' \
+    && apt-get update -y && apt-get install -y clang-11 \
+    && wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - \
+    && add-apt-repository 'deb http://apt.llvm.org/focal/ llvm-toolchain-focal-13 main' \
+    && apt-get update -y && apt-get install -y clang-13
+
+# `futures` is needed for `yapf` parallelization on Python 2.
```

Comment:
I guess this isn't true/needed since we don't have yapf for python2 anymore?

Reply:
Deleted `futures`, rebuilt the container, and then built unit tests and python bindings with python2 and everything worked fine so I think I can get rid of it.